scheduler/node/node.go

```diff
@@ -0,0 +1,86 @@
+package node
```

Comment:
yes, but `scheduler` imports the strategies and filters, so we cannot use scheduler.Node in the strategies and filters